hls4ml/templates/quartus/myproject_test_stream.cpp

```diff
@@ -0,0 +1,128 @@
+#include <fstream>
+#include <iostream>
+#include <algorithm>
+#include <vector>
+#include <string>
+#include <cctype>
+
+#include "firmware/parameters.h"
+#include "firmware/myproject.h"
+
+#include "firmware/nnet_utils/nnet_helpers.h"
+
+#define CHECKPOINT 5000
+
+// This function is written to avoid stringstream, which is
+// not supported in cosim 20.1, and because strtok
+// requires a const_cast or allocation to use with std::strings.
+// This function returns the next float (by argument) at position pos,
+// updating pos. True is returned if conversion done, false if the string
+// has ended, and std::invalid_argument exception if the sting was bad.
+bool nextToken(const std::string& str, std::size_t& pos, float& val) {
+  while (pos < str.size() && std::isspace(static_cast<unsigned char>(str[pos]))) {
+    pos++;
+  }
+  if (pos >= str.size()) {
+    return false;
+  }
+  std::size_t offset = 0;
+  val = std::stof(str.substr(pos), &offset);
+  pos += offset;
+  return true;
+}
+
+int main(int argc, char **argv) {
```

Comment:
Can we use 4 spaces for indent in the testbench files, in line with the other source files?

Reply:
Done.